content/blog/2023/03/10/2023-03-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,149 @@
+- The link:/blog/2023/02/23/cdf-awards/[Jenkins contributor awards] by author:alyssat[Alyssa Tong]
+- Multiple posts regarding link:/blog/2023/02/01/gsoc-update/[Google Summer of Code] and link:/blog/2023/02/23/gsoc2023-announcement/[our participation] from author:jmmeessen[Jean-Marc Meessen]
+- Our link:/blog/2023/02/07/jenkins-newsletter/[January newsletter] from the Jenkins SIG leaders
+- A link:/blog/2023/02/28/build-msi-locally/[tutorial on building a Jenkins MSI] on a Windows machine by author:gounthar[Bruno Verachten]
+
+We've also received numerous pull requests from contributors that are getting started with Jenkins, as well as several excited participants of the Google Summer of Code.
+For all of the work and energy you're putting into the Jenkins project, we extend our deepest gratitude.
+
+[[Governance]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/governance.png[Governance Update]
+
+Contributed by: author:markewaite[Mark Waite]
+
+The Jenkins governance board met once in February, resolved several action items, and noted the progress on projects with sponsors like JFrog and Atlassian.
+We're sincerely grateful for the sponsorships provided by those generous companies and many other companies.
+
+[[platform]]
+image:/images/post-images/2023/01/12/jenkins-newsletter/platform-modernization.png[Platform Modernization Update]
+
+Contributed by: author:gounthar[Bruno Verachten]
+
+Ongoing work:
+
+ - Centos `7` Jenkins Controller Docker Image proposal: deprecate the centos images.
+We'll need an announcement and a https://github.com/jenkinsci/jep[JEP].
+We'll also release a merged version of the `centos` and `centos7` image as the very last centos `7` Docker image.
+
+Docker images:
+
+* Latest updates on the agent images:
+** ssh-agent: bump Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-ssh-agent/pull/209[#209]) @github-actions
+** docker-agent: bump Git version on Windows to 2.39.2.windows.1 (https://github.com/jenkinsci/docker-agent/pull/376[#376]) @github-actions
+** inbound agent:
+*** bump the parent image jenkins/agent version to 3107.v665000b_51092-4 (https://github.com/jenkinsci/docker-inbound-agent/pull/331[#331]) @github-actions
+*** bump the parent image jenkins/agent version to 3107.v665000b_51092-3 (https://github.com/jenkinsci/docker-inbound-agent/pull/330[#330]) @github-actions
+*** bump updatecli/updatecli-action from 2.19.0 to 2.20.1 (https://github.com/jenkinsci/docker-inbound-agent/pull/329[#329]) https://github.com/dependabot[@dependabot]
+*** Windows controller image is not updated as often as the rest. It's been more than one year without any update. We may simply drop it.
+
+* Debian 12 (“bookworm”) will not deliver OpenJDK 11
+** The end of life date for Debian's openJDK11 won't happen until 2026 or 2027.
+** The documentation will be updated when it goes out, so that we describe the use of Jenkins with openJDK17.
```

Comment:
```suggestion
* With the release of Debian 12 (“bookworm”), Debian will no longer deliver OpenJDK 11.
** Thankfully, the end of life date for Debian's openJDK11 won't happen until 2026 or 2027.
** The Jenkins documentation will be updated when it goes out, so that we describe the use and installation of Jenkins with openJDK17.
```
adding some words/sentence syntax